### PR TITLE
Fix Layout: Textview was on top of Switch

### PR DESCRIPTION
**Pull request by pascaldornfeld**
The textview was on top of scrollview

### CODE REVIEW

**Comment by pascaldornfeld** [comment]
Before device-2020-04-06-135015
After device-2020-04-06-140548

**Comment by Nicba1010** [comment]
We could possibly consider expanding the actual item so that the user can see the whole name he assigned.

**Comment by pascaldornfeld** [comment]
There you go

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -1,33 +1,6 @@
...
-      <formatting-settings enabled="false">
-        <option name="STATEMENT_SPACING" value="one_line" />
-        <option name="CLAUSE_CHOP_DOWN" value="chop_down_if_statement_long" />
-        <option name="ITERATION_ELEMENTS_WRAPPING" value="chop_down_if_not_single" />
-      </formatting-settings>
-    </DBN-SQL>
     <codeStyleSettings language="XML">
-      <indentOptions>
-        <option name="CONTINUATION_INDENT_SIZE" value="4" />
-      </indentOptions>
```
Comment (borna-mdn) [inline .idea/codeStyles/Project.xml:30]: Why did you commit this file?

File: .idea/modules.xml [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
+      <module fileurl="file://$PROJECT_DIR$/app/Sensorics-app.iml" filepath="$PROJECT_DIR$/app/Sensorics-app.iml" group="Sensorics/app" />
+      <module fileurl="file://$PROJECT_DIR$/data/Sensorics-data.iml" filepath="$PROJECT_DIR$/data/Sensorics-data.iml" group="Sensorics/data" />
+      <module fileurl="file://$PROJECT_DIR$/device/Sensorics-device.iml" filepath="$PROJECT_DIR$/device/Sensorics-device.iml" group="Sensorics/device" />
+      <module fileurl="file://$PROJECT_DIR$/domain/Sensorics-domain.iml" filepath="$PROJECT_DIR$/domain/Sensorics-domain.iml" group="Sensorics/domain" />
+      <module fileurl="file://$PROJECT_DIR$/StudioProjects-Sensorics.iml" filepath="$PROJECT_DIR$/StudioProjects-Sensorics.iml" group="Sensorics" />
     </modules>
   </component>
-</project>
\ No newline at end of file
+</project>
```
Comment (borna-mdn) [inline .idea/modules.xml:12]: Why did you commit this change??

File: app/build.gradle [hunk 3]
```diff
@@ -126,7 +126,7 @@ dependencies {
             ".inject:assisted-inject-annotations-dagger2:$assisted_inject_dagger_version"
     kapt "com.squareup.inject:assisted-inject-processor-dagger2:$assisted_inject_dagger_version"
 
-    debugImplementation "com.squareup.leakcanary:leakcanary-android:$leak_canary_version"
+   // debugImplementation "com.squareup.leakcanary:leakcanary-android:$leak_canary_version"
```
Comment (borna-mdn) [inline app/build.gradle:129]: Why did you disable Leakcanary??

**Comment by pascaldornfeld** [comment]
Applied the same changes to item of virtualScanningSource.
Removed unnecessary files from last commit.

**Comment by borna-mdn** [comment]
Why is this WIP?

**Comment by pascaldornfeld** [comment]
Other items may have similar problems but I decided not to fix them here because I can't test them. It can be merged.

**Pull request merged**